var/spack/repos/builtin/packages/vmc/package.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Vmc(CMakePackage):
+    """The Virtual Monte Carlo (VMC) library"""
+
+    homepage = "https://github.com/vmc-project/vmc"
+    git      = 'https://github.com/vmc-project/vmc.git'
+    url      = "https://github.com/vmc-project/vmc/archive/v1-0.tar.gz"
+
+    maintainers = ['ChristianTackeGSI']
+
+    version('1-0-p2', sha256='46b4c82b0b7516502e88db920732fc78f06f0393ac740a17816f2eb53f80e75e')
+    version('1-0-p1', sha256='4a20515f7de426797955cec4a271958b07afbaa330770eeefb5805c882ad9749')
+
+    patch('dict_fixes_101.patch', when='@1-0-p1')
+
+    depends_on('root@6.18.04: ~vmc')
+
+    def common_env_setup(self, env):
+        # So that root finds the shared library / rootmap
+        env.prepend_path("LD_LIBRARY_PATH", self.prefix.lib)
+
+    def setup_run_environment(self, env):
+        self.common_env_setup(env)
```

Comment:
Spack already adds this by default, this shouldn't be needed.

Reply:
It was needed in older spack versions. I have used this recipe already for a while, just now upstreaming it.

Removed. Let's see.

(Note: I think, some parts need improvement: After removing all the `setup*` a `spack load vmc; spack unload vmc` still leaves some stuff in `$LD_LIBRARY_PATH`.)